### DISCUSSION TITLE
Get forecast in raw, STANDARD JSON?

### DISCUSSION THREAD

**Discussion by macccorehenni** [weather-gov/api#716] [Q&A] [answered]
I'm trying to make a widget for my phone's Lock Screen that I can see the next forecast segment there. However, I have one issue with this— when I try and access the /points API endpoint, it only gives me a link to the point forecast on weather.gov. Is there any way I could request plain JSON rather than JSON? Am I using the wrong endpoint? If so, what should I use?

**Comment by dwhitemv25** [comment]
What's the API URL you're using?

**Reply by macccorehenni** [reply]
> What's the API URL you're using?

https://api.weather.gov/points/{latitude},{longitude}

**Reply by dwhitemv25** [reply]
That query should be giving you geo+json output if the lat/lon is valid. What is the exact output of your query? Does it work if you take the same URL the app is fetching and paste it into a web browser?

Things I'm thinking of as common issues:

* Truncate the coordinates to 4 decimal places. It will throw you a redirect if you aren't. 
* Coordinates must be in degrees. Longitude must be in the range (-180,180) and latitude (-90, 90).
* Send a valid Accept: header. `application/geo+json` is assumed if you don't send one, but if your app is sending something random the API might respond strangely to it.

**Reply by macccorehenni** [reply]
Is there a way I can have it return standard JSON?

**Reply by dwhitemv25** [reply]
What's "standard JSON"? geo+json is a standard.

**Reply by macccorehenni** [reply]
A MIME type of text/json.

**Reply by dwhitemv25** [reply, marked as answer]
No, the API does not emit bare `text/json`. The supported output formats are listed under "Formats" on https://www.weather.gov/documentation/services-web-api, on the Overview tab. 

`application/geo+json` is a JSON output with fields in particular places, commonly used with Web GIS applications. The forecast shows up as a property of the 'feature' (point). Anything that can read JSON should have no trouble parsing it.
